SIPS/sip-12.md

```diff
@@ -13,7 +13,7 @@ This SIP proposes a new endowment, `endowment:name-lookup`, that enables a way f
 
 ## Motivation
 
-Currently, the MetaMask wallet allows for ENS domain resolution. The implementation is hardcoded and limited to just the ENS protocol. In an effort to increasingly modularize the wallet and allow for resolution beyond ENS, we decided to open up domain/address resolution to snaps. A snap would be able to provide resolution based on a domain or address provided with a chain ID. The address resolution is in essence "reverse resolution". The functionality provided by this API is also beneficial as a base layer for a petname system (**see definition**). With plans to bring petnames to MetaMask, resolutions would be fed into the petname system and used as a means for cache invalidation.
+Currently, the MetaMask wallet allows for ENS domain resolution. The implementation is hardcoded and limited to just the ENS protocol. In an effort to increasingly modularize the wallet and allow for resolution beyond ENS, we decided to open up domain/address resolution to snaps. A snap would be able to provide resolutions based on a domain or address provided with a chain ID. The address resolution is in essence "reverse resolution". The functionality provided by this API is also beneficial as a base layer for a petname system (**see definition**). With plans to bring petnames to MetaMask, resolutions would be fed into the petname system and used as a means for cache invalidation.
```

Comment:
```suggestion
Currently, the MetaMask wallet allows for ENS domain resolution. 
The implementation is hardcoded and limited to just the ENS protocol. 
In an effort to increasingly modularize the wallet and allow for resolution beyond ENS, we decided to open up domain/address resolution to snaps. 
A snap would be able to provide resolutions based on a domain or address provided with a chain ID. 
The address resolution is in essence "reverse resolution". 
The functionality provided by this API is also beneficial as a base layer for a petname system (**see definition**). 
With plans to bring petnames to MetaMask, resolutions would be fed into the petname system and used as a means for cache invalidation.
```

newlines make diffs easier to read, particularly for the `vim` sickos

Reply:
Done!